plugins/content_distribution/lib/api/KalturaDistributionProfile.php

```diff
@@ -153,7 +153,15 @@ abstract class KalturaDistributionProfile extends KalturaObject implements IFilt
 	 * @var int
 	 */
 	public $recommendedDcForExecute;
-	
+
+
+	/**
+	 * The event that trigger the automatic distribute
+	 * @var KalturaDistributeOnType
+	 */
+	public $distributeOnType;
```

Comment:
Better change from "DistributeOnType" to "DistributionTrigger" same goes to the enum and admin console changes